code/lib/telemetry/src/storybook-metadata.test.ts

```diff
@@ -82,26 +82,85 @@ describe('sanitizeAddonName', () => {
     // @ts-expect-error the property is read only but we can change it for testing purposes
     path.sep = '\\';
     const cwdMockPath = `C:\\Users\\username\\storybook-app`;
-    jest.spyOn(process, `cwd`).mockImplementationOnce(() => cwdMockPath);
+    const cwdSpy = jest.spyOn(process, `cwd`).mockReturnValue(cwdMockPath);
 
     expect(sanitizeAddonName(`${cwdMockPath}\\local-addon\\themes.js`)).toEqual(
       '$SNIP\\local-addon\\themes'
     );
+    cwdSpy.mockRestore();
```

Comment:
1. Do we need this if `cwdSpy` is a local variable?
2. If so, is it guaranteed that this line will execute given that there could be an error on one of the previous lines?

Reply:
Good one, I changed the code in a way that it will always restore the mocks even if it errors